streamer/src/nonblocking/stream_throttle.rs

```diff
@@ -150,6 +150,15 @@ impl StakedStreamLoadEMA {
         peer_type: ConnectionPeerType,
         total_stake: u64,
     ) -> u64 {
+        // If it is a staked connection with ultra low stake ratio, treat it as unstaked.
+        if let ConnectionPeerType::Staked(stake) = &peer_type {
```

Comment:
wouldn't it be easier to make the decision here, rather than worry about whether we need to reconsider everywhere?

https://github.com/anza-xyz/agave/blob/dd8e1f4c7311eae0e8864ea87fa2a72b2f2fbdcd/streamer/src/nonblocking/quic.rs#L503-L507

Reply:
Good idea. Done